test/schedule.test.ts

```
@@ -1,21 +1,18 @@
 
-describe('test/schedule.test.js', () => {
-  let app;
+const mm = _mm.default;
+
+describe('test/schedule.test.ts', () => {
+  let app: MockApplication;
   afterEach(() => app.close());
 
   describe('schedule type worker', () => {
-    it('should support interval and cron', async () => {
+    it.only('should support interval and cron', async () => {
```

Biome (1.9.4)

[error] Don't focus the test.

The 'only' method is often used for debugging or during implementation. It should be removed before deploying to production.
Consider removing 'only' to ensure all tests are executed.
Unsafe fix: Remove focus from test.


(lint/suspicious/noFocusedTests)